Lesson34/selcan/lesson34-app/src/components/UserDetailsForm/index.jsx

```diff
@@ -0,0 +1,25 @@
+import { useState } from "react";
+import "./styles.css";
+
+// TODO: Update component so that it's possible to change user name and save new user name in the UserContext
+
+export const UserDetailsForm = ({ incomingUsername }) => {
+  const [username, setUsername] = useState("");
+
+  function handleSubmit(e) {
+    e.preventDefault();
+    console.log("New user name is: ", username);
+  }
+
+  return (
+    <form className="user-details-form" onSubmit={handleSubmit}>
```

Comment:
Make sure to style this form so the input element is visible on the page. Right now it's completely white on the white background, it took me some time to find the form.